programs/uxd/src/state/controller.rs

```diff
@@ -94,6 +101,13 @@ pub struct Controller {
     pub mercurial_vault_depository: Pubkey,
     pub credix_lp_depository: Pubkey,
 
+    // Flags needed for outflow limitation
+    pub outflow_limit_per_epoch_amount: u64,
+    pub outflow_limit_per_epoch_bps: u16,
+    pub seconds_per_epoch: u32,
+    pub epoch_outflow_amount: u64,
+    pub last_outflow_timestamp: i64,
```

Comment:
Maybe `epoch_start_timestamp`?

Reply:
Looking into the comments, I double checked and I think this name make semantic sense:
`last_outflow_timestamp` is the timestamp of the last redeem, it is not the start of the epoch (the epoch start is constantly X seconds ago, we dont need to store that I believe)

We just need this flag to compute "how long ago was the last redeem" so that we know how much we can decrease the current `epoch_outflow_amount` by.

Maybe i'm miss understanding your comment tho 👌 please let me know